cmd/root.go

```diff
@@ -7,8 +7,8 @@ import (
 // rootCmd represents the base command when called without any subcommands.
 var rootCmd = &cobra.Command{
 	Use:   "locode-db",
-	Short: "NeoFS UN/LOCODE database CLI",
-	Long:  `locode db CLI is a tool for working with NeoFS UN/LOCODE database.`,
+	Short: "UN/LOCODE database CLI",
```

Comment:
it was always bothering me tbh. we do not _generate UN/LOCODE_ database, we download it and extend it. while it was _NeoFS_ UN/LOCODE database i could accept it a little bit easier but now it sounds stranger to me
@roman-khimov

Reply:
I don't think it's critical. Maybe some wording can be improved in future, but current one is not incorrect.